src/inmanta/config.py

```diff
@@ -95,6 +96,8 @@ def load_config(
         files: list[str]
         if min_c_config_file is not None:
             files = [main_cfg_file] + cfg_files_in_config_dir + local_dot_inmanta_cfg_files + [min_c_config_file]
+            cls._min_c_config_file = min_c_config_file
```

Comment:
We must reset this value in the `reset()` method.

Reply:
We should also set this value explicitly to `None` in the else-block. For the situation where `reset()` wasn't called first.